README.md

```diff
@@ -461,6 +461,17 @@ var consentManager = SPConsentManager(
 )
 ```
 
+## Keeping support to legacy CCPA privacy string
+
+If you're transitioning from CCPA to USNat, odds are you want to keep supporting the legacy CCPA privacy string (`IABUSPrivacy_String`).
+To do so, when instantiating the SDK, make sure to set the flag `.supportLegacyUSPString` to true. Example:
+```swift
+var consentManager = SPConsentManager(
+    ...
+    campaigns: SPCampaigns(usnat: SPCampaign(supportLegacyUSPString: true)), // <== here
+    delegate: self
+)
+```
```

Comment:
@tom-win87 the section above relates to supporting USP string via USnat campaigns. We may want to have something similar in our docs. Let me know if you need more context 🦜

Reply:
I just made very small edits to what you wrote in a commit

Reply:
Appreciate you.